lib/connect-backbone-to-react.js

```diff
@@ -112,6 +115,17 @@ module.exports = function connectBackboneToReact(
       componentWillReceiveProps(nextProps, nextContext) {
         this.setModels(nextProps, nextContext);
         this.createNewProps();
+
+        // Bind event listeners for each model that changed.
+        Object.keys(this.models).forEach(mapKey => {
+          const model = this.models[mapKey];
+          if (this.props.models[mapKey] === this.models[mapKey] ||
```

Comment:
We've already done the lookup on `this.models[mapKey]` so let's just use the `model` variable here and in the next line.

Also should we be comparing against `nextProps.models`? As that's the new incoming model?

Reply:
Sorry for the delay on this! I just pushed a change to DRY up the key lookup.

On `nextProps.models` - it's set as `this.models` by `setModels` by the time this code runs. So we are comparing against `nextProps.models` (or `nextContext.models`, if that's used instead).

Reply:
ping @hswolff

Reply:
Aha, got it.